core/hmholtz.f

```diff
@@ -3364,9 +3364,9 @@ subroutine axhelm_acc_debug (au,u,helm1,helm2,imesh,isd)
      $ ,             tmp2  (lx1*ly1*lz1*lelt)
      $ ,             tmp3  (lx1*ly1*lz1*lelt)
 
-      real           tm1   (lx1*ly1*lz1,lelt)
-      real           tm2   (lx1*ly1*lz1,lelt)
-      real           tm3   (lx1*ly1*lz1,lelt)
+      real           tm1   (lx1*ly1*lz1*lelt)
```

Comment:
I believe this is the intended dimensionality.  The previous declaration:
`real tm1 (lx1*ly1*lz1, lelt)`
failed at line 3525.